README.md

```diff
@@ -184,6 +209,20 @@ ip-172-XX-XX-XX.ec2.internal  node  True
 
 Your cluster <HOSTED_ZONE> is ready
 ```
+From this single command a large number of AWS resources are provisioned. The command creates six subnets in the VPC. Three public, three private, and four route tables to go along with them. One public route table with three subnet associations, and one route table for each of the private subnets.
+
+Each private subnet also has it's own associated NAT Gateway. That seems excessive, but I imagine there's a reason for it. Each NAT Gateway is associated with a public Elastic IP Address. The command creates several security groups to permit communication between the instances.
+
+Also, kops creates five instances. One "master", three "nodes", and a bastion host. The bastion host has a public IP address, but the master and nodes in the Kubernetes cluster do not. The bastion host allows only SSH access (port 22) from the security group "sg-0b1b434d07c25e1c2", which is associated with a classic ELB. The ELB listens on port 22 and is open to the world.
```

Comment:
Where does `sg-0b1b434d07c25e1c2` come from? I imagine that will be different for the person who re-deploys this?

Reply:
True, I could re-word this a bit to make it more generic.